data/docs_markdown/Monitor_Block_Rewards.md

```
@@ -0,0 +1,108 @@
+# Monitor Block Rewards
+## [](https://docs.io.net/docs/monitor-block-rewards#table-of-contents)
+  * [Block Rewards Tab](https://docs.io.net/docs/monitor-block-rewards#block-rewards-tab)
+  * [Block Details Table](https://docs.io.net/docs/monitor-block-rewards#block-details-table)
+  * [Block Reward Details](https://docs.io.net/docs/monitor-block-rewards#block-reward-details)
+    * [Exceptions](https://docs.io.net/docs/monitor-block-rewards#exceptions)
+
+
+### [](https://docs.io.net/docs/monitor-block-rewards#block-rewards-tab)
+The Block Rewards tab provides a transparent view of io.net's Block Rewards and coin emissions. Users can consult this information to monitor worker nominations and their status. Users can track the performance and success rates for worker nominations and block completion. Information on coin emissions and block rewards provides transparency about io.net network’s health.
+The screenshot below is of the **Block Rewards** tab and is explained the sections below. 
+![](https://files.readme.io/a4a3f9b-block_rewards.png)
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content

(MD024, no-duplicate-heading)